proto/internalapi/sensor/compliance_nodescan.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
```

Comment:
About the file name `compliance_nodescan.proto`. I think, the fact that we put Scanner in Compliance is just an implementation detail that we shouldn't promote to the contract. As a thought exercise, `compliance_` part will become misleading if we decide to set up a dedicated DaemonSet for node scanner.

Reply:
I think it would make more sense to actually put this in `proto/storage/node.proto` next to `Node` and the older `NodeScan`. That way we can store it in Central once all the vulns are filled in, and we can use it in other places as well. It also decouples it from the internal API. I believe `internalapi/...` can import from `storage/`, right?